apps/cowswap-frontend/src/modules/limitOrders/containers/SettingsWidget/index.tsx

```diff
@@ -1,71 +1,29 @@
 import { useAtomValue, useSetAtom } from 'jotai'
-import React, { useCallback, useState } from 'react'
+import React from 'react'
 
 import { Menu, MenuItem } from '@reach/menu-button'
 
-import { ExpertModeIndicator, MenuContent, SettingsButton, SettingsIcon } from 'modules/trade/pure/Settings'
-
-import { ExpertModeModal } from 'common/pure/ExpertModeModal'
+import { MenuContent, SettingsButton, SettingsIcon } from 'modules/trade/pure/Settings'
 
 import { Settings } from '../../pure/Settings'
-import {
-  limitOrdersSettingsAtom,
-  LimitOrdersSettingsState,
-  updateLimitOrdersSettingsAtom,
-} from '../../state/limitOrdersSettingsAtom'
+import { limitOrdersSettingsAtom, updateLimitOrdersSettingsAtom } from '../../state/limitOrdersSettingsAtom'
 
 export function SettingsWidget() {
   const settingsState = useAtomValue(limitOrdersSettingsAtom)
   const updateSettingsState = useSetAtom(updateLimitOrdersSettingsAtom)
-  const [showExpertConfirm, setShowExpertConfirm] = useState(false)
-
-  const onStateChanged = useCallback(
-    (state: Partial<LimitOrdersSettingsState>) => {
-      const { expertMode } = state
-
-      if (expertMode !== undefined) {
-        const isExpertModeOn = !settingsState.expertMode && expertMode
-        const isExpertModeOff = settingsState.expertMode && !expertMode
-
-        if (isExpertModeOn) {
-          setShowExpertConfirm(true)
-        } else if (isExpertModeOff) {
-          updateSettingsState({ expertMode: false })
-        }
-      } else {
-        updateSettingsState(state)
-      }
-    },
-    [settingsState, updateSettingsState]
-  )
-  const onEnableExpertMode = useCallback(() => {
-    updateSettingsState({ expertMode: true })
-    setShowExpertConfirm(false)
-  }, [updateSettingsState, setShowExpertConfirm])
 
   return (
     <>
       <Menu>
         <SettingsButton>
           <SettingsIcon />
-          {settingsState.expertMode && (
-            <ExpertModeIndicator>
-              <span>🐮</span>
-              <span>🥋</span>
```

Comment:
Nooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo


Kimono CoW! This is a sad day

Reply:
![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fwww.icegif.com%2Fwp-content%2Fuploads%2Ficegif-2830.gif&f=1&nofb=1&ipt=3ec53792f67fcfe9eb1674a81bdcd765ce651a46e066d2d5d9e866c4401a3a4f&ipo=images)